mw-webapp/src/component/usersBlock/UsersBlock.tsx

```diff
@@ -17,7 +17,8 @@ export const UsersBlock = () => {
   }, []);
 
   const renderUsers = () => {
-    return users.map((user) => (
+    const usersObjectToArray = Object.values(users);
+    return usersObjectToArray.map((user) => (
       <UserCard key={user.uuid}
```

Comment:
Please think about naming. Inside function renderUsers we work with users so I think better use const users for map it. You can change naming for state const